docs/API-functions.md

```diff
@@ -2057,9 +2051,92 @@ will provide this information on its own.
 
 The following flags are available, as collected in `logging_api.h` public header:
 
-* `SRT_LOGF_DISABLE_TIME`: Do not provide the time in the header
-* `SRT_LOGF_DISABLE_THREADNAME`: Do not provide the thread name in the header
-* `SRT_LOGF_DISABLE_SEVERITY`: Do not provide severity information in the header
-* `SRT_LOGF_DISABLE_EOL`: Do not add the end-of-line character to the log line
+- `SRT_LOGF_DISABLE_TIME`: Do not provide the time in the header
+- `SRT_LOGF_DISABLE_THREADNAME`: Do not provide the thread name in the header
+- `SRT_LOGF_DISABLE_SEVERITY`: Do not provide severity information in the header
+- `SRT_LOGF_DISABLE_EOL`: Do not add the end-of-line character to the log line
+
+## Time Access
+
+The following set of functions is intended to retrieve timestamps from clock used by SRT.
+This timestamp MUST be used by sender if it wants to explicitely assign a timestamp
+to a certain packet. The sender can pass the timestamp in `MSGCTRL::srctime` of `srt_sendmsg2(..)`
+function together with the packet being submitted to SRT.
+The time provided MUST equal or exceed connection start time (`srt_connection_time(..)`)
+of the SRT socket passed to `srt_sendmsg2(..)`.
+
+The current time value as of SRT internal clock can be retrieved using `srt_time_now()` function.
+
+There are two known use cases of why you might want to use `srctime`:
```

Comment:
```suggestion
There are two known cases where you might want to use `srctime`:
```

Reply:
The term "use cases" was there for a reason. Maybe "There are two known use cases for `srctime`"?

Reply:
I think "cases" are also good.

Reply:
The word "use" in the same sentence qualifies "case".